selinux/ceph.te

```diff
@@ -119,6 +120,13 @@ nis_use_ypbind_uncond(ceph_t)
 storage_raw_rw_fixed_disk(ceph_t)
 files_manage_generic_locks(ceph_t)
 libs_exec_ldconfig(ceph_t)
+fs_list_hugetlbfs(ceph_t)
+fs_list_tmpfs(ceph_t)
+fs_read_cgroup_files(ceph_t)
+fs_read_tmpfs_symlinks(ceph_t)
+fs_search_cgroup_dirs(ceph_t)
+init_nnp_daemon_domain(ceph_t)
```

Comment:
```
ceph.te:128:ERROR 'syntax error' at token 'init_nnp_daemon_domain' on line 8624:
init_nnp_daemon_domain(ceph_t)
 	
/usr/bin/checkmodule:  error(s) encountered while parsing configuration
/usr/bin/checkmodule:  loading policy configuration from tmp/ceph.tmp
make[3]: *** [tmp/ceph.mod] Error 1
make[2]: *** [selinux/CMakeFiles/ceph.pp] Error 2
make[1]: *** [selinux/CMakeFiles/ceph.pp.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

guess this depends on RHEL 8 policy settings. see https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=centos7,DIST=centos7,MACHINE_SIZE=huge/314//consoleFull